### ISSUE TITLE
Demo BLEU in the spanish-english translation example

### ISSUE THREAD

**Issue by mattdangerw** [keras-team/keras-nlp#259]
Once we land the BLEU metric, let's add BLEU to the mix of metrics we are showing off here:
https://keras.io/examples/nlp/neural_machine_translation_with_keras_nlp/#evaluating-our-model-quantitative-analysis

**Comment by mattdangerw** [comment]
@abheesht17 does this make sense to you? Would you like to add this as a minor improvement?

**Comment by abheesht17** [comment]
It does! Will add it :)
